OptimizelySDKCore/OptimizelySDKCore.xcodeproj/project.pbxproj

```diff
@@ -192,8 +192,8 @@
 		EA16D93D1ECBD90E00C4C998 /* OPTLYExperimentBucketMapEntity.h in Headers */ = {isa = PBXBuildFile; fileRef = EA16D93A1ECBD90E00C4C998 /* OPTLYExperimentBucketMapEntity.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		EA16D93E1ECBD90E00C4C998 /* OPTLYExperimentBucketMapEntity.m in Sources */ = {isa = PBXBuildFile; fileRef = EA16D93B1ECBD90E00C4C998 /* OPTLYExperimentBucketMapEntity.m */; };
 		EA16D93F1ECBD90E00C4C998 /* OPTLYExperimentBucketMapEntity.m in Sources */ = {isa = PBXBuildFile; fileRef = EA16D93B1ECBD90E00C4C998 /* OPTLYExperimentBucketMapEntity.m */; };
-		EA16D9421ECCC5C600C4C998 /* OPTLYDecisionService.h in Headers */ = {isa = PBXBuildFile; fileRef = EA16D9401ECCC5C600C4C998 /* OPTLYDecisionService.h */; };
-		EA16D9431ECCC5C600C4C998 /* OPTLYDecisionService.h in Headers */ = {isa = PBXBuildFile; fileRef = EA16D9401ECCC5C600C4C998 /* OPTLYDecisionService.h */; };
+		EA16D9421ECCC5C600C4C998 /* OPTLYDecisionService.h in Headers */ = {isa = PBXBuildFile; fileRef = EA16D9401ECCC5C600C4C998 /* OPTLYDecisionService.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
why does this need to be public?

Reply:
Here is the compiler warning a customer would
see following the README.md "Manual Instructions" if we did
not make OPTLYDecisionService.h be 'Public':

<module-includes>:1:1: warning: umbrella header for module 'OptimizelySDKiOS' does not include header 'OPTLYDecisionService.h' [-Wincomplete-umbrella]
#import "Headers/OptimizelySDKiOS.h"
^
1 warning generated.
1 warning generated.

Reply:
does it need to be in the umbrella? The customer never personally interacts with it right?